pkg/output/stdout.go

```diff
@@ -159,6 +163,11 @@ func (s *Stdoutput) resultNormal(resp ffuf.Response) {
 	fmt.Println(res_str)
 }
 
+func (s *Stdoutput) resultAlt(resp ffuf.Response) {
+	res_str := fmt.Sprintf("%s %-3s %-9d %-5d %s", TERMINAL_CLEAR_LINE, s.colorizeStatus(resp.StatusCode), resp.ContentLength, resp.ContentWords, resp.Url)
```

Comment:
I'd like to keep descriptive output in the default operation, so newcomers would be able to easily figure out which value is which. Alternatively a "table header" line for output would suffice with this kind of default output. I mean something like:

```
Status |   Size   | Words | Input
-----------------------------------
   201     1024        64   https://ffuf.io.fi/matched.result
   200     1315       124   https://ffuf.io.fi/another.result
```
and obviously when `FUZZ` keyword wasn't in the URL;

```
Status |   Size   | Words | Input
-----------------------------------
   201     1024        64   matched.result
   200     1315       124   another.result
```


Please let me know what you think. This doesn't have to happen in this PR, and I can implement that, but I'd like to hear your opinion nevertheless.